lib/addExcursionPointers.m

```diff
@@ -0,0 +1,132 @@
+%==========================================================================
+%This function takes in an NIDM-Results json and creates a hashmap labelling
+%objects with a pointer to their relevant excursion set. The resultant
+%hashmap takes in the id of an object from the main NIDM-Results graph and
+%outputs a number for it's corresponding ExcursionMap.
+%
+%graph - the graph from an NIDM-Results json.
+%
+%Authors: Thomas Maullin, Camille Maumet.
+%==========================================================================
+
+function labels = addExcursionPointers(graph)
+    
+    %Create an empty list for the keys and values.
+    labelValues = {};
+    labelKeys = {};
+    
```

Comment:
I'm a little confused as to what you mean here - this function creates a map of that form - but to do so must first create a list of keys and a list of values associated to the keys - this is what the addExcursionPointers is doing here (the map is created at line 130)